doc/source/reference/routines.other.rst

```diff
@@ -35,3 +36,33 @@ NumPy version comparison
    :toctree: generated/
 
    lib.NumpyVersion
+
+NumPy header files location
+----------------------------
+.. autosummary::
+   :toctree: generated/
+   
+   get_include
+
+Deprecation warning
+-------------------
+.. autosummary::
+   :toctree: generated/
+
+   deprecate
+
+Display message
+---------------
+.. autosummary::
+   :toctree: generated/
+
+   disp
+
+Details of used variables
+-------------------------
+.. autosummary::
+   :toctree: generated/
+ 
+   who
```

Comment:
Could ```'who'``` be nested under [Data type routines](http://www.numpy.org/devdocs/reference/routines.dtype.html#miscellaneous)?

Reply:
Data types are the information about what is inside the array given by `arr.dtype`, `who` lists arrays.